src/utils/helpers.js

```diff
@@ -269,10 +269,21 @@ const helpers = {
    * @returns {Promise<ReduxTxHistory>}
    */
   async mapTxHistoryToRedux(wallet, tx, tokenUid) {
-    // tx comes from getTxHistory and does not have token_data
-    // We need the actual history tx to access if it is an authority tx
-    const histTx = await wallet.getTx(tx.txId);
-    const isAllAuthority = this.isAllAuthority(histTx);
+
+    let isAllAuthority = false;
+    try {
+      // tx comes from getTxHistory and does not have token_data
+      // We need the actual history tx to access if it is an authority tx
+      const histTx = await wallet.getTx(tx.txId);
```

Comment:
note: wallet-service facade does not implement `getTx`, it throws when called.

This was wrapped in a try..catch and `isAllAuthority` will default to false in this case.

Reply:
I agree, and I believe that we should check this and other methods to have parity on the facades.

For instance, just implementing `getTx` would be bad since it uses an api call for each tx and the wallet facade gets it from memory (O(1) operation) since we use the `getTx` sequentially for each tx we want to show on screen it would severely impact the refresh rate.